mac/bashHelperFunctions.sh

```diff
@@ -0,0 +1,40 @@
+# The following allows coloring of warning and error lines, but only works if there's a
```

Comment:
I can (and will) add the #!/bin/sh notation, but when I do "git add --chmod=+x bashHelperFunctions.sh" at the command line, it doesn't register as a change. Am I doing something wrong? (Locally, it already shows that it is executable.)